update_maker/update_maker.py

```diff
@@ -45,12 +50,24 @@ def __init__(self, board: str):
         Path(DOWNLOAD_DIR).mkdir(parents=True, exist_ok=True)
         Path(UPDATE_CACHE_DIR).mkdir(parents=True, exist_ok=True)
         Path(STATUS_CACHE_DIR).mkdir(parents=True, exist_ok=True)
+        Path(OLU_SIGNATURES_DIR).mkdir(parents=True, exist_ok=True)
 
         # clear download dir
         for i in listdir(DOWNLOAD_DIR):
             if i.endswith(".deb"):
                 remove(DOWNLOAD_DIR + i)
 
+        # copying the context of the real root apt source.list file into the local one
+        copyfile(SYSTEM_APT_SOURCES_FILE, OLU_APT_SOURCES_FILE)    
+
+        # adding OreSat Debian apt repo
+        with open(OLU_APT_SOURCES_FILE, 'a') as f:
+            f.write('deb [trusted=yes] https://debian.oresat.org/packages ./') 
+            f.close() 
+
+        # copying the apt repo signatures
+        system(f'cp {SYSTEM_SIGNATURES_DIR}* {OLU_SIGNATURES_DIR}')  
```

Comment:
Don't use system(), as it is consider a bad practice.

Maybe use  listdir() and call copyfile() in a loop.

Reply:
Ok. I'll change it and make another commit.

Reply:
Also, now that I'm think about it, might want all of this to happen before cache is initialize and add a check `source.list` file exists. Therefor update_maker doesn't override all the files each time the program is started.

Reply:
Okay. Probably it all will be ready by Monday.